### DISCUSSION TITLE
Proposal for Steering Committee Compensation

### DISCUSSION THREAD

**Discussion by casinoyoda** [ringdao-community-guild/discussion#97] [Proposals Temperature Check]
## Summary

This proposal outlines the plan to offer monetary compensation to the members of the Steering Committee of the Darwinia Community DAO (DCDAO) for their dedication, preparation, and consistent efforts in facilitating the growth and governance of the Darwinia community. The proposed compensation is set at a total of 1,000 USD equivalent in RING tokens per month, to be evenly distributed among the five committee members.

## Introduction

The Darwinia Community DAO (DCDAO) serves as a pivotal self-organized workgroup (WG) that plays a crucial role in nurturing and advancing the Darwinia network's ecosystem. The Steering Committee, as the executive arm of the DCDAO, is tasked with the regular orchestration of weekly meetings, the meticulous review of proposals and forum discussions, the strategic coordination of resources, and the provision of expert recommendations for the refinement and advancement of community projects.

Recognizing the commitment and time invested by the Steering Committee members, it is proposed that a financial incentive be awarded to acknowledge their contributions and to attract talented individuals who are motivated to guide the Darwinia community towards its strategic objectives.

### Compensation Structure

The compensation for the Steering Committee members of the DCDAO is proposed as follows:

- A monthly compensation pool of 1,000 USD equivalent in RING tokens.
- The pool will be divided equally among the five members of the Steering Committee, resulting in 200 USD equivalent in RING tokens per member per month.
- This compensation is intended to cover the members' preparation, efforts, and attendance in the weekly meetings.

### Duration of Compensation

The compensation is proposed to start from Q3 2024 and will be evaluated on an annual basis to ensure alignment with the DCDAO's objectives and financial capabilities.

### Motivation for Compensation

The rationale behind providing compensation to the Steering Committee members includes:

- Acknowledging the significant time and effort invested by the members in the governance and development of the community.
- Enhancing the motivation and commitment of the members by providing a tangible recognition of their contributions.
- Attracting skilled and dedicated individuals to the Steering Committee who can offer valuable insights and leadership.
- Ensuring the sustainability and effectiveness of the DCDAO governance structure by rewarding diligent service.

### Funding

The compensation will be funded by the DCDAO treasury, subject to the approval of this proposal by the community. The treasury is tasked with the management and allocation of funds in a manner that supports the community's growth and governance.

## Conclusion

The proposed compensation for the Steering Committee members is designed to acknowledge their invaluable contributions to the DCDAO and to ensure that the committee continues to attract and retain individuals capable of steering the community towards a prosperous future. It is with great anticipation that we submit this proposal for the community's consideration and look forward to the continued success and evolution of the Darwinia network.

---

This proposal is open for discussion, and we invite all members of the Darwinia Community DAO to provide their feedback and suggestions. Your participation is crucial for the democratic governance and progressive development of our community.

Here are some related links:
- [Apply for a position on the Steering Committee for the next term](https://github.com/orgs/dcdao/discussions/81)
- [About the Steering Committee](https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x849cf36a644c2c2162bf79b518c288817e64b536ea42338be1e094fd574fe069)

**Comment by SasoLithops** [comment]
I absolutely support this.

**Comment by 2xJDubs** [comment]
I support this proposal as well and will happily vote for it.

**Comment by casinoyoda** [comment]
https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x1e279f76e243b5e3d7c8be80f7e8373c7084d1a5cbe2222012a1208442ba4ba5
